src/wallet/wallet.cpp

```diff
@@ -1777,6 +1784,14 @@ bool CWallet::ImportScriptPubKeys(const std::string& label, const std::set<CScri
     return true;
 }
 
+void CWallet::FirstKeyTimeChanged(const ScriptPubKeyMan* spkm, int64_t new_birth_time)
+{
+    int64_t birthtime = m_birth_time.load();
+    if (birthtime == BLANK_BIRTH_TIME || new_birth_time < birthtime) {
+        m_birth_time = new_birth_time;
+    }
```

Comment:
Because the other one is a return statement (the flow breaks there) while this one is a setter.
Plus, when the line is too long, like this one, I think that braces help readability.